frontend/test/metabase/scenarios/models/models-metadata.cy.spec.js

```diff
@@ -134,8 +134,9 @@ describe("scenarios > models metadata", () => {
     cy.button("Save changes").click();
 
     // Revision 1
-    cy.findByText("Subtotal ($)");
-    cy.findByText("Tax ($)").should("not.exist");
```

Comment:
@kulyk Seeing the search for a formatted tax header threw me off for a couple of minutes, tbh. Although the assertion is passing, it's confusing to see it before we ever attempted to remap the Tax column.

I've updated the assertion. Hope this makes sense to you?

Reply:
> Seeing the search for a formatted tax header threw me off for a couple of minutes, tbh. Although the assertion is passing, it's confusing to see it before we ever attempted to remap the Tax column.

@nemanjaglumac not sure I follow; as far as I can see the mapping is done _before_ the assertions. See the [line remapping the "Subtotal" column](https://github.com/metabase/metabase/blob/390ebb73f76c7369b5d95c655472fd90878db922/frontend/test/metabase/scenarios/models/models-metadata.cy.spec.js#L132) and [the one mapping the "Tax" column](https://github.com/metabase/metabase/blob/390ebb73f76c7369b5d95c655472fd90878db922/frontend/test/metabase/scenarios/models/models-metadata.cy.spec.js#L149). Is that what you meant?

The change looks good to me, thank you!